proto/kvrpcpb.proto

```diff
@@ -142,9 +142,9 @@ enum Op {
 }
 
 enum Assertion {
-     None = 0;
-     Exist = 1;
-     NotExist = 2;
+    None = 0;
```

Comment:
It's better to define `UNKNOWN = 0`, https://developers.google.com/protocol-buffers/docs/proto3#enum

Reply:
ok

Reply:
@overvenus hey, do i need to modify its name in this PR?

Reply:
Yes, better to follow the style, also please address the [comment](https://github.com/pingcap/kvproto/pull/414/files#r300267740), change tag ID is not allowed since it's a breaking change.